### ISSUE TITLE
bulk rpc 

### ISSUE THREAD

**Issue by alexandregiordanelli** [supabase/supabase#1710]
# Bug report
problem on bulk rpc
## Describe the bug
not showing the data of each object of array requested

## To Reproduce
```bash
curl 'https://dtcoquimlibtwkzuzjgu.supabase.co/rest/v1/rpc/getobjectid3' \
  -H 'apikey: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYyMTg5NjE5MiwiZXhwIjoxOTM3NDcyMTkyfQ.JHLYOdsxVkW62sjwXGMJKdfYDC4YxpcSYKAIb6LQj3E' \
  -H 'content-type: application/json' \
  --data-raw '[
  {
    "user_uuid": "42c12e46-426f-4c58-9b49-189ccf41f84f",
    "name_object": "42c12e46-426f-4c58-9b49-189ccf41f84f/1.png"
  },
  {
    "user_uuid": "42c12e46-426f-4c58-9b49-189ccf41f84f",
    "name_object": "42c12e46-426f-4c58-9b49-189ccf41f84f/bom.png"
  }
]'
```

## Expected behavior

["60fce85b-ace3-434f-9a86-d36775417c0e", "a86d76b7-91a6-4dde-9ffb-a8cb5003e2a9"]

but the response was:
["60fce85b-ace3-434f-9a86-d36775417c0e"]

## Screenshots

<img width="894" alt="Captura de Tela 2021-05-25 às 19 34 18" src="https://user-images.githubusercontent.com/1518223/119577272-ae2c2100-bd90-11eb-97ae-7964413fb518.png">
<img width="901" alt="Captura de Tela 2021-05-25 às 19 34 35" src="https://user-images.githubusercontent.com/1518223/119577274-aff5e480-bd90-11eb-97dc-d8e7d9188e2a.png">
<img width="576" alt="Captura de Tela 2021-05-25 às 19 38 33" src="https://user-images.githubusercontent.com/1518223/119577339-d6b41b00-bd90-11eb-8756-9b1cdbae0e75.png">

## Additional context
I found this issue because I just like to receive the object_id when I upload a file, but we receive the key of object (bucker + name)

**Comment by steve-chavez** [comment]
Try adding the `Prefer: params=multiple-objects` header.

```bash
curl -H 'Prefer: params=multiple-objects'...
```

More details at https://postgrest.org/en/stable/api.html#bulk-call

**Comment by alexandregiordanelli** [comment]
Wow.. It works, but the supabase js client does not put this header. 
<img width="519" alt="Captura de Tela 2021-05-25 às 19 56 13" src="https://user-images.githubusercontent.com/1518223/119578643-49be9100-bd93-11eb-92cd-060795c50ea3.png">

https://supabase.io/docs/reference/javascript/rpc#bulk-call

**Comment by steve-chavez** [comment]
> but the supabase js client does not put this header.

Yeah, true. Could you make your `getobjectid3` function use an array instead of many parameters? That would work with the current `supabase-js`.

**Comment by alexandregiordanelli** [comment]
Sure, I created a workaround. Do I close this issue?
